Sources/PostgresKit/PostgresConfiguration.swift

```diff
@@ -49,7 +52,9 @@ public struct PostgresConfiguration {
         username: String,
         password: String,
         database: String? = nil,
-        tlsConfiguration: TLSConfiguration? = nil
+        tlsConfiguration: TLSConfiguration? = nil,
+        jsonEncoder: JSONEncoder = JSONEncoder(),
```

Comment:
This can accept `PostgresDataEncoder` / `Decoder` since those have public init methods.